### DISCUSSION TITLE
Docs don't specify how to change savefile path for modded game - maybe have root path that is inheritted?

### DISCUSSION THREAD

**Discussion by knzai** [exult/exult#525] [General]
The docs show how to change the paths for an unmodded game and talk about how the path changes when modded. But there is nothing I can find that specifies how to change that path to somewhere more user accessible (many Mac games at this point use Documents instead of App Support, and that is synced which is nice) for the modded game specifically. 

I've tried guessing at keynames in the xml and am about to do a code dive.

Would it maybe make sense to have \<default\> option under \<game\> for setting the root paths (or other defaults) and then the usual logic can apply and you don't have to tweak each game (and/or figure out how to try and get modded games working with it)?

Either that or a section in the config docs near the path part that mentions how to set paths for modded games/saves?

**Comment by DominusExult** [comment]
I think for now it may be possible to change the path in the mod‘s cfg.

a general path for user generated data is not a bad idea ;)

To still have the saves synced you could create an Exult folder in ~/Documents and create a symlink to that in ~/Library/Application Support

That‘s how I sync my files on macOS via Dropbox. Game folders and generated data in the Dropbox folder and symlinks to /Library/Application Support/Exult and ~/Library/Application Support/Exult

**Comment by knzai** [comment]
Yeah, that's what I usually end up doing.

I do try to nudge things towards needing less tweaks during setup to fit common usage patterns though also. The path generation macros and code definitely would take me refreshing on modern C/C++ to successfully post a PR though. My first look was a reminder how much I've forgotten.
